pkg/kubelet/kubelet_pods.go

```diff
@@ -2068,3 +2068,13 @@ func (kl *Kubelet) hasHostMountPVC(pod *v1.Pod) bool {
 	}
 	return false
 }
+
+func isAbs(path string) bool {
```

Comment:
Same comment as before. If this is an issue with the golang, please file a bug with them and reference this bug here. I have hard time to believe that this issue only affects k8s.